src/protocols/rdp/guac_rdpdr/rdpdr_printer.h

```diff
@@ -25,44 +25,32 @@
 
 #include "rdpdr_service.h"
 
-#include <guacamole/stream.h>
+#include <guacamole/user.h>
```

Comment:
Nothing. It's a remnant of a part of the change which ended up not being necessary. Will remove.